Framework/DataObjects/src/EventList.cpp

```diff
@@ -3937,6 +3953,31 @@ void EventList::filterInPlace(Kernel::SplittingIntervalVec &splitter) {
   }
 }
 
+/**
+ * Initialize the detector ID's and event type of the destination event lists when splitting this list.
+ * @param partials : resulting partial lists of events after splitting's done
+ */
+void EventList::initializePartials(std::map<int, EventList *> partials) const {
+
+  // collect the state from events which is to be transferred to the partials
+  bool removeDetIDs{true};
+  const auto &detIDs = this->getDetectorIDs();
+  auto histogram = this->getHistogram();
+  auto eventType = this->getEventType();
+
+  // lambda expression initializing one partial
+  auto initPartial = [&](EventList *partial) {
+    partial->clear(removeDetIDs);
+    partial->copyInfoFrom(*this);
+    partial->setHistogram(histogram); // problem here!
```

Comment:
Is this a spurious comment (please remove) or something that needs to be addressed/expanded on?

Reply:
yes, there was an issue here some time ago but now it's fixed :smile: